src/pages/CreateSession/index.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect } from "react";
+import Form from "react-bootstrap/Form";
+import Col from "react-bootstrap/Col";
+import Button from "react-bootstrap/Button";
+import { selectUser } from "../../store/user/selectors";
+import { useHistory } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { Jumbotron } from "react-bootstrap";
+import { selectSubject } from "../../store/subject/selectors";
+import { fetchSubjects } from "../../store/subject/actions";
+import { createSession } from "../../store/session/actions";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+export default function CreateSessionForm() {
+  const { token } = useSelector(selectUser);
+  const user = useSelector(selectUser);
+  const subjects = useSelector(selectSubject);
+
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [start_date, setStart_date] = useState(new Date());
```

Comment:
same comment on the `snake_case` as below, for vars and function names, be consistent!